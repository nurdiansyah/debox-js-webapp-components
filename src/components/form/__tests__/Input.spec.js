import React from 'react'

import {Form, Input, getFormBlock, getInputBlock} from '../index'

describe('form/Input', () => {
  it('should running no error', () => {
    const formBlock = getFormBlock({})
    const inputBlock = getInputBlock({})
    const wrapper = render(
      <Form block={formBlock}>
        <Input block={inputBlock} name="buah" />
      </Form>
    )
    expect(wrapper).toMatchSnapshot()
  })
})
