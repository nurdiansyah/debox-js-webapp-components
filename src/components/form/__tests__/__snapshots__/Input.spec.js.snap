// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form/Input should running no error 1`] = `
<form
  class="form"
>
  <div
    class="form__control"
  >
    <input
      class="input"
      id="buah1"
      name="buah"
      value=""
    />
  </div>
</form>
`;
